composer/loggers/logger.py

```diff
@@ -79,18 +79,18 @@ def __init__(
         self.destinations = ensure_tuple(destinations)
         self._state = state
 
-    def data(self, log_level: Union[str, int, LogLevel], data: Dict[str, Any]) -> None:
-        """Log data to the :attr:`destinations`.
+    def log_traces(self, traces: Dict[str, Any]):
+        for destination in self.destinations:
+            destination.log_traces(traces)
```

Comment:
Why does this have to be different from metrics?

Reply:
It's for logging [engine traces,](https://github.com/mosaicml/composer/blob/dev/composer/core/engine.py#L310) which we want in the `FileLogger`, but we don't want in `WandBLogger`, so `WandBLogger` doesn't implement `log_traces`